library/src/main/java/org/xmtp/android/library/SigningKey.kt

```diff
@@ -18,13 +18,16 @@ import java.util.Date
 
 interface SigningKey {
     val address: String
-    /// If this signing key is a smart contract wallet
+
+    // If this signing key is a smart contract wallet
```

Comment:
This was actually stopping the build for me due to a ktLint error, the rest is just auto-formatting

Reply:
Sorry about that!